CONTRIBUTING.md

```diff
@@ -55,3 +59,46 @@ your local DynamoDB instance, hit:
 ```sh
 USE_DYNAMODB=local cargo run
 ```
+
+### API Gateway Local
+
+Prerequisites:
```

Comment:
Maybe mention here that API Gateway will only work when the binary is built in release mode (which SAM does)?

Reply:
Sure thing

Reply:
Added a note on that just a little bit lower